live-build/misc/ansible-roles/appliance-build.unittest-internal/tasks/main.yml

```diff
@@ -23,6 +23,7 @@
       - build-essential
       - docker.io
       - git
+      - gdisk
```

Comment:
Since this in YML, is it possible to add a code comment here explaining why this is needed. I think that might help us identify it as a potential cleanup if we ever find we don't need that `enable-swap.sh` script